internal/gitlab/gitlab.go

```diff
@@ -888,3 +888,10 @@ func Labels(labels []string) *gitlab.Labels {
 	l := gitlab.Labels(labels)
 	return &l
 }
+
+// TextToGitlabText converts text with Gitlab friendly line breaks.
+// See https://gist.github.com/shaunlebron/746476e6e7a4d698b373 for more info.
+func TextToGitlabText(text string) string {
```

Comment:
This isn't really gitlab specific behavior, it's just markdown, so on the topic of naming I don't think this is named correctly, nor in the right package. I don't really have a better suggestion tho, should of just being an unexported function in the `cmd` package...

https://markdown-guide.readthedocs.io/en/latest/basics.html#line-return

Reply:
Y'know when I add the --comment flag to 'mr show' I noticed that the 'issue show' and 'mr show' commands had VERY similar structure and was wondering if there was a common place for lab code.  I would have unified some of the 'mr show' and 'issue show' code.

I haven't seen one?  If there isn't one perhaps creating "cmd/common.go" is a good idea.  I'm sure there are other functions that could be made common.

What do you think?  The code base is eventually going to need a common area.

Reply:
I just pushed a new version with a cmd/common.go file (and the other fixes discussed above) to see what you think.

Reply:
To date, I've put common stuff in `cmd/root.go`, but we probably should have made a `cmd/common.go` or `cmd/util.go` a while ago.

Reply:
Let me look at root.go and think about util.go.  I do think a util.go is the way to go.

Reply:
I've pushed a util.go and made the other changes.  Let me know what you think.  I'll put in a test for util.go as well.